internal/platform/wrap_windows.go

```diff
@@ -61,7 +62,11 @@ func (w *windowsWrappedFile) Write(p []byte) (n int, err error) {
 		return
 	}
 
-	return w.WriteFile.Write(p)
+	n, err = w.WriteFile.Write(p)
+	if errors.Is(err, ERROR_ACCESS_DENIED) {
```

Comment:
ok so this is the change huh
```suggestion
	// ERROR_ACCESS_DENIED is often returned instead of EBADF
	// when a file is used after close.
	if errors.Is(err, ERROR_ACCESS_DENIED) {
```